FirebaseMessaging.podspec

```diff
@@ -35,7 +35,8 @@ device, and it is completely free.
 
   base_dir = "FirebaseMessaging/"
   s.source_files = [
-    base_dir + 'Sources/**/*.[mh]',
```

Comment:
`base_dir + 'Sources/**/*.[mhc]',` would be safer I believe

Reply:
Done!